locations/spiders/ica_se.py

```diff
@@ -0,0 +1,13 @@
+from scrapy.spiders import SitemapSpider
+
+from locations.categories import Categories, apply_category
+from locations.structured_data_spider import StructuredDataSpider
+
+
+class IcaSESpider(SitemapSpider, StructuredDataSpider):
+    name = "ica_se"
+    item_attributes = {"brand": "ICA", "brand_wikidata": "Q1663776", "extras": Categories.SHOP_SUPERMARKET.value}
+    sitemap_urls = ["https://www.ica.se/butiker/sitemaps/1/", "https://www.ica.se/butiker/sitemap.xml"]
```

Comment:
Instead of `https://www.ica.se/butiker/sitemaps/1/` you can use `https://www.ica.se/butiker/sitemaps/`, then when/if `https://www.ica.se/butiker/sitemaps/2/` comes along, we'll get it too.

```suggestion
    sitemap_urls = ["https://www.ica.se/butiker/sitemaps/", "https://www.ica.se/butiker/sitemap.xml"]
```